docs/tutorials/image_signing.md

```diff
@@ -0,0 +1,189 @@
+# How to sign and distribute container images using Podman
+
+Signing container images originates from the motivation of trusting only
+dedicated image providers to mitigate man-in-the-middle (MITM) attacks. One way
+to sign images is to utilize a GNU Privacy Guard ([GPG][0]) key. This technique
+is generally compatible with any OCI compliant container registry like
+[Quay.io][1]. It is worth mentioning that the OpenShift integrated container
+registry supports this signing mechanism out of the box.
+
+[0]: https://gnupg.org
+[1]: https://quay.io
+
+From a technical perspective, we can utilize Podman to sign the image before
+pushing it into a remote registry. After that, all systems running Podman have
+to be configured to retrieve the signature key from a remote server, which can
+be any simple web server. This means that every unsigned image will be rejected
+during an image pull operation. But how does this work?
+
+First of all, we have to create a GPG key pair or select an already locally
+available one:
```

Comment:
I don't know if you've one handy, but maybe a quick para on how to create a gpg key or a pointer to a blog about the same?

Reply:
FWIW, IDK if it's the best, but I've used: https://www.linuxbabe.com/security/a-practical-guide-to-gpg-part-1-generate-your-keypair

Reply:
Alright, I added a sentence about how to generate the key. :+1: